provisioning/ansible/config/profiles/beetbox.config.yml

```diff
@@ -11,7 +11,6 @@ installed_extras_apparmor: yes
 installed_extras_avahi: yes
 installed_extras_composer: yes
 installed_extras_drush: yes
-installed_extras_drupal_console: yes
```

Comment:
It's now recommended to add as a project dependency + https://github.com/geerlingguy/drupal-vm/pull/1339#issuecomment-296872882